web/client/components/map/openlayers/__tests__/Layer-test.jsx

```diff
@@ -850,6 +850,43 @@ describe('Openlayers layer', () => {
         expect(map.getLayers().getLength()).toBe(1);
     });
 
+    it('creates a vector layer with opacity for openlayers map', () => {
+        const opacity = 0.45;
+        var options = {
+            crs: 'EPSG:4326',
+            opacity,
+            features: {
+                'type': 'FeatureCollection',
+                'crs': {
+                    'type': 'name',
+                    'properties': {
+                        'name': 'EPSG:4326'
+                    }
+                },
+                'features': [
+                    {
+                        'type': 'Feature',
+                        'geometry': {
+                            'type': 'Polygon',
+                            'coordinates': [[
+                              [13, 43],
+                              [15, 43],
+                              [15, 44],
+                              [13, 44]
+                            ]]
+                        }
+                    }
+                ]
+            }
+        };
+        // create layers
+        var layer = ReactDOM.render(
+            <OpenlayersLayer type="vector"
+                 options={options} map={map}/>, document.getElementById("container"));
+
+        expect(layer.layer.N.opacity).toEqual(opacity);
```

Comment:
This tests the compiled version... Is possible to get instead the opactity using API methods? I see the [original PR](https://github.com/geosolutions-it/MapStore2/pull/4231/files#diff-9a06e1d40abd2ef3ac80ae0d43fa62e9R1041) checked a `values_`, maybe a better approch (for both master and this PR) is to use `getOpacity()` is it possible?